src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx

```diff
@@ -302,4 +302,119 @@ Here's the response:
 ```
 
 
+## Browser Agent Version Pinning [#browser-agent-version-pinning]
 
+New Relic's GraphQL API provides a method to "pin" a specific version of the New Relic Browser Agent, ensuring it remains consistent within a customer's platform. By pinning a version, you can prevent automatic updates that might introduce unexpected changes or behaviors. Key benefits of using this feature include:
+
+- **Control**: Retain autonomy over when and how updates are applied.
+- **Confidence**: Ensure that a tested and approved version of the agent is running at all times.
+- **Testability**: Easily test new versions in isolated environments before deciding to update.
+- **Stability**: Minimize unexpected disruptions and maintain consistent application behavior.
+- **Efficiency**: Reduce deploy time and mitigate deployment difficulties.
+
+Here is an overview of how to use the Browser Agent Version Pinning API:
+
+1. Check agent type and which version is currently pinned, if any
+1. Reference the available versions of the agent
+1. Select and pin the desired version
+1. Redeploy or restart your service to use the pinned version of the agent
```

Comment:
> Do we need to add the decoding step that appears down in the main steps?

I think, that in the case of step 4 and 5 being combined as I suggested in the other comment, it would be okay to not include the decoding step in the overview.

Reply:
Removed step 4. Decode the JavaScript snippet